test/onnx_import.cpp

```diff
@@ -413,3 +413,52 @@ TEST(onnx, model_softmax)
     auto result_vectors = execute(function, inputs, "INTERPRETER");
     EXPECT_TRUE(test::all_close_f(expected_output, result_vectors.front()));
 }
+
+TEST(onnx, model_flatten)
+{
+    auto function = onnx_import::import_onnx_function(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/flatten.onnx"));
+
+    Inputs inputs;
+
+    inputs.emplace_back(
+        test::NDArray<float, 4>(
+            {{{{1, 2, 3, 4, 5}, {6, 7, 8, 9, 10}, {11, 12, 13, 14, 15}, {16, 17, 18, 19, 20}},
+              {{21, 22, 23, 24, 25},
+               {26, 27, 28, 29, 30},
+               {31, 32, 33, 34, 35},
+               {36, 37, 38, 39, 40}},
+              {{41, 42, 43, 44, 45},
+               {46, 47, 48, 49, 50},
+               {51, 52, 53, 54, 55},
+               {56, 57, 58, 59, 60}}},
+
+             {{{61, 62, 63, 64, 65},
+               {66, 67, 68, 69, 70},
+               {71, 72, 73, 74, 75},
+               {76, 77, 78, 79, 80}},
+              {{81, 82, 83, 84, 85},
+               {86, 87, 88, 89, 90},
+               {91, 92, 93, 94, 95},
+               {96, 97, 98, 99, 100}},
+              {{101, 102, 103, 104, 105},
+               {106, 107, 108, 109, 110},
+               {111, 112, 113, 114, 115},
+               {116, 117, 118, 119, 120}}}})
+            .get_vector());
+
+    auto expected_output =
+        test::NDArray<float, 2>(
+            {{1,   2,   3,   4,   5,   6,   7,   8,   9,   10,  11,  12,  13,  14,  15,
+              16,  17,  18,  19,  20,  21,  22,  23,  24,  25,  26,  27,  28,  29,  30,
```

Comment:
Please append `.f` to every float value.

Reply:
There is a NDArray of dtype=float inicialized by int values like in other tests.

Reply:
This is C++, you need `.f`, the compiler will assume in this case `double` and not `float`.

Reply:
https://github.com/NervanaSystems/ngraph/blob/master/test/backend_test.in.cpp <- Acording to this file there is no need to .f appended to int values.